src/newLaunch/lbp/stages/stage4.ts

```diff
@@ -108,48 +110,48 @@ export class Stage4 extends BaseStage {
     const target = event.target as HTMLInputElement;
     const value = parseInt(target.value);
 
-    if (value > this.lbpConfig.lbpDetails.startWeight) {
-      target.value = this.lbpConfig.lbpDetails.startWeight.toString();
+    if (value > this.launchConfig.lbpDetails.startWeight) {
+      target.value = this.launchConfig.lbpDetails.startWeight.toString();
     }
-    this.lbpConfig.lbpDetails.endWeight = value;
+    this.launchConfig.lbpDetails.endWeight = value;
   }
 
-  setLbpConfigStartDate(): Date {
+  setlaunchConfigStartDate(): Date {
     // Set the ISO time
     // Get the start and end time
     const startTimes = this.startTime.split(":");
     const temp = this.startDate;
     temp.setHours(Number.parseInt(startTimes[0]), Number.parseInt(startTimes[1]));
-    this.lbpConfig.lbpDetails.startDate = this.dateService.toISOString(this.dateService.translateLocalToUtc(temp));
-    return new Date(this.lbpConfig.lbpDetails.startDate);
+    this.launchConfig.lbpDetails.startDate = this.dateService.toISOString(this.dateService.translateLocalToUtc(temp));
+    return new Date(this.launchConfig.lbpDetails.startDate);
   }
 
-  setLbpConfigEndDate(): Date {
+  setlaunchConfigEndDate(): Date {
     // Set the ISO time
     // Get the start and end time
     const endTimes = this.endTime.split(":");
     const temp = this.endDate;
     temp.setHours(Number.parseInt(endTimes[0]), Number.parseInt(endTimes[1]));
-    this.lbpConfig.lbpDetails.endDate = this.dateService.toISOString(this.dateService.translateLocalToUtc(temp));
-    return new Date(this.lbpConfig.lbpDetails.endDate);
+    this.launchConfig.lbpDetails.endDate = this.dateService.toISOString(this.dateService.translateLocalToUtc(temp));
+    return new Date(this.launchConfig.lbpDetails.endDate);
   }
 
   // persistData(): void {
-  //   this.setLbpConfigStartDate();
-  //   this.setLbpConfigEndDate();
-  //   this.wizardState.lbpStartDate = this.lbpConfig.lbpDetails.startDate;
+  //   this.setlaunchConfigStartDate();
+  //   this.setlaunchConfigEndDate();
+  //   this.wizardState.lbpStartDate = this.launchConfig.lbpDetails.startDate;
   // }
 
   toggleLegalDisclaimer(): void {
-    this.lbpConfig.lbpDetails.legalDisclaimer = !this.lbpConfig.lbpDetails.legalDisclaimer;
+    this.launchConfig.lbpDetails.legalDisclaimer = !this.launchConfig.lbpDetails.legalDisclaimer;
```

Comment:
following from my suggestion above, this would be `this.launchConfig.launchDetails`

Reply:
implemented in 596f1f0